src/parser/transform/expression/transform_function.cpp

```diff
@@ -299,6 +299,8 @@ unique_ptr<ParsedExpression> Transformer::TransformFuncCall(duckdb_libpgquery::P
 			                                               std::move(filter_expr), std::move(order_bys),
 			                                               root->agg_distinct, false, root->export_state);
 			lowercase_name = "list_sort";
+			order_bys.reset();
```

Comment:
Why not having reset performed (potentially implicitly) by std::move?

I think that the problem might be that std::move is not taking a duckdb::unique_ptr (and it's also not returning a duckdb::unique_ptr), but narrowing down to std::unique_ptr.

Reply:
Sure I thought about that, but how?
I don't think there's a specific method invoked by `std::move`
Only on the receiving part right?

Oh actually maybe in the move constructor/assignment operator I can call reset on the lhs

Reply:
> I think that the problem might be that std::move is not taking a duckdb::unique_ptr (and it's also not returning a duckdb::unique_ptr), but narrowing down to std::unique_ptr.

I don't think that's true?
it takes X and produces X, why would it downcast anything?